Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -173,7 +173,7 @@ private func localizedDescription(project: Project, reward: Reward) -> String {
 }
 
 private func rewardTitle(project: Project, reward: Reward) -> NSAttributedString {
-  guard project.personalization.isBacking == true else {
+  guard project.personalization.isBacking == true || currentUserIsCreator(of: project) else {
```

Comment:
This `guard` was preventing the reward title from being formatted correctly in the creator context.